### PR TITLE
Release 1.1.2

### PR DESCRIPTION
**Pull request by Technius**
This adds changelog entries for the commits since 1.1.1.

Edit (Oct 15): also add last remaining changes for v1.1.2.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+* Fixed a bug where currently editing text boxes would not unfocus when moving
+  to a different page (#3765, #4027).
+* Fixed stylus/mouse inputs being detected twice for some tools (#4162).
+* Fixed a crash that occurs when a plugin calls `app.getDocumentStructure()`
+  (#4192, #4196).
+* Fixed a memory leak that occurs when using the Stroke Recognizer (#4258).
+* Improved performance of Text Tool rendering by eliminating unnecessary redraws
+  (#3810, #4000, #4074, #4135, #4136).
+* Improved LaTeX Tool error message when running inside Flatpak by providing
+  steps on how to install the TeXLive extension (#3996, #4259).
```
Comment (bhennion) [inline CHANGELOG.md:21]:
```suggestion
  steps on how to install the TeXLive extension (#3996, #4259).
* Prevent potentiel segmentation fault in toolBar customization (#4293)
```

**Comment by Technius** [comment]
@xournalpp/core Please double check the changelog one last time. Thanks in advance.

**Review by bhennion — APPROVED** [review]

**Review by rolandlo — COMMENTED** [review]
Looks like e2f7f0f562645f4c971b43b6e44dd2dc39c61763 (PR #4297) is missing from the list.

About the macOS change:
- permission issues are only avoided if the binary is signed, which it currently isn't (although a user could self-sign it). Removing the launcher script was only the first step into that direction.
- Since https://github.com/xournalpp/xournalpp/pull/4153 is not finished, a user with 2-letter language codes will be faced with a crash, see https://github.com/xournalpp/xournalpp/pull/4153#pullrequestreview-1041295535

As mentioned in the gitter chat, we could backport https://github.com/xournalpp/xournalpp/pull/4228/files fixing the attached mode.

Otherwise everything looks fine to me.

**Comment by Technius** [comment]
I'm repurposing this PR to incorporate the changes required for the 1.1.2 release.
* Updated changelog with the missing commits, please check one last time (promise)!
* Cherry-picked the 2-letter language code fix for mac in #4153
* Backported #4228
* Update version numbers

**Comment by Technius** [comment]
Testing the release pipeline once... (version will be `1.1.3~dev` but that's to be expected).

/azp run create-installers

**Comment by LittleHuba** [comment]
The requested installers were generated successfully. The installers are available at:

- [Windows](https://artprodsu6weu.artifacts.visualstudio.com/A4bb312fe-95d7-456e-96fb-7669bb27c94b/ce85de88-4b71-44a0-8a57-2707c0b9a6e7/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3hvdXJuYWxwcC9wcm9qZWN0SWQvY2U4NWRlODgtNGI3MS00NGEwLThhNTctMjcwN2MwYjlhNmU3L2J1aWxkSWQvMjE3NDIvYXJ0aWZhY3ROYW1lL3dpbmRvd3M1/content?format=zip)
- [macOS](https://artprodsu6weu.artifacts.visualstudio.com/A4bb312fe-95d7-456e-96fb-7669bb27c94b/ce85de88-4b71-44a0-8a57-2707c0b9a6e7/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3hvdXJuYWxwcC9wcm9qZWN0SWQvY2U4NWRlODgtNGI3MS00NGEwLThhNTctMjcwN2MwYjlhNmU3L2J1aWxkSWQvMjE3NDIvYXJ0aWZhY3ROYW1lL21hY09T0/content?format=zip)
- [Debian](https://artprodsu6weu.artifacts.visualstudio.com/A4bb312fe-95d7-456e-96fb-7669bb27c94b/ce85de88-4b71-44a0-8a57-2707c0b9a6e7/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3hvdXJuYWxwcC9wcm9qZWN0SWQvY2U4NWRlODgtNGI3MS00NGEwLThhNTctMjcwN2MwYjlhNmU3L2J1aWxkSWQvMjE3NDIvYXJ0aWZhY3ROYW1lL2xpbnV4LWRlYmlhbi1wYWNrYWdlcw2/content?format=zip)
- [Ubuntu 18.04](https://artprodsu6weu.artifacts.visualstudio.com/A4bb312fe-95d7-456e-96fb-7669bb27c94b/ce85de88-4b71-44a0-8a57-2707c0b9a6e7/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3hvdXJuYWxwcC9wcm9qZWN0SWQvY2U4NWRlODgtNGI3MS00NGEwLThhNTctMjcwN2MwYjlhNmU3L2J1aWxkSWQvMjE3NDIvYXJ0aWZhY3ROYW1lL2xpbnV4LXVidW50dS0xOC4wNC1wYWNrYWdlcw2/content?format=zip)
- [Ubuntu 20.04](https://artprodsu6weu.artifacts.visualstudio.com/A4bb312fe-95d7-456e-96fb-7669bb27c94b/ce85de88-4b71-44a0-8a57-2707c0b9a6e7/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3hvdXJuYWxwcC9wcm9qZWN0SWQvY2U4NWRlODgtNGI3MS00NGEwLThhNTctMjcwN2MwYjlhNmU3L2J1aWxkSWQvMjE3NDIvYXJ0aWZhY3ROYW1lL2xpbnV4LXVidW50dS0yMC4wNC1wYWNrYWdlcw2/content?format=zip)

Please be aware that these links are only valid for a limited time (~30 days).

**Comment by Rmano** [comment]
Thanks! Just a comment: will the release have a windows-portable (#3714) version? I could not find it in the https://github.com/xournalpp/xournalpp/pull/4287#issuecomment-1279695207 artifacts...

**Comment by rolandlo** [comment]
It looks all good to me, except of the automated version number bump (last commit). I have tested on Ubuntu 22.10 (built from source and via the AppImage), on MacOS Catalina and on Windows 10.

I found one issue on Windows 10 (no filename is suggested when saving a new file), but that is also present in version 1.1.1 and on master.

**Comment by rolandlo** [comment]
> Thanks! Just a comment: will the release have a windows-portable (#3714) version? I could not find it in the [#4287 (comment)](https://github.com/xournalpp/xournalpp/pull/4287#issuecomment-1279695207) artifacts...

I think not, since the corresponding changes have been merged into `master`, not into `release-1.1` branch. See #3842

**Comment by Rmano** [comment]
> think not, since the corresponding changes have been merged into `master`, not into `release-1.1` branch. See #3842

Oh. A pity, I hoped from https://github.com/xournalpp/xournalpp/pull/3842#issuecomment-1046311339 that it would... having it would be a useful option for students (and non-students...) with "locked" laptops.

**Comment by Technius** [comment]
I backported the portable zip step from #3842, thanks for the catch.

**Comment by Technius** [comment]
I forgot to rebase on the latest changes in release-1.1, did so just now.

**Pull request merged**